packages/react-utils/tsup.config.ts

```diff
@@ -12,6 +12,7 @@ export default defineConfig({
   esbuildPlugins: [
     preserveDirectivesPlugin({
       directives: ['use client', 'use strict'],
+      // eslint-disable-next-line prefer-named-capture-group -- Is this a necessary rule?
```

Comment:
Great question

Reply:
I'll leave the eslint-disable as is for now